doc/glossary.rst

```diff
@@ -76,13 +76,31 @@ MNE-Python core terminology and general concepts
         down to a change of the ``first_samp`` attribute to know when cropped data
         was acquired.
 
+    forward model
```

Comment:
```suggestion
    forward solution
```

Reply:
@agramfort are these terms actually different? I thought they were basically interchangable. There are [places in the C API reference](https://mne-tools.github.io/dev/manual/c_reference.html#mne-convert-mne-data) where both "forward model" and "forward solution" are used in the text describing different command-line flags of the same function.

I guess if you want to standardize, there's a simple argument for going with "forward solution":
```bash
$ git grep "forward model" | wc -l
51
$ git grep "forward solution" | wc -l
252
```

Reply:
Let's do "forward solution" for now and find-and-replace in a quick follow-up PR

Reply:
we need a model of the head, this is the BEM or sphere geometrical model. Then given the head model, we solve maxwell's equations and the solution of the computation is the "forward solution". Let's stick to BEM model and forward solution.